src/lib/split-view/split-view-panel/split-view-panel-adapter.ts

```diff
@@ -0,0 +1,211 @@
+import { getCursor, getHandleIcon, getSplitViewPanelSibling } from '../core/split-view-core-utils';
+import { ISplitViewPanelComponent } from './split-view-panel';
+import { SplitViewPanelPosition, SPLIT_VIEW_PANEL_CONSTANTS } from './split-view-panel-constants';
+import { SplitViewOrientation, SPLIT_VIEW_CONSTANTS } from '../split-view/split-view-constants';
+import { ISplitViewComponent } from '../split-view/split-view';
+import { IIconComponent } from '../../icon';
+
+export interface ISplitViewPanelAdapter extends IBaseAdapter {
+  initialize(): void;
+  setPointerdownListener(listener: (evt: PointerEvent) => void): void;
+  removePointerdownListener(listener: (evt: PointerEvent) => void): void;
+  setPointerupListener(listener: (evt: PointerEvent) => void): void;
+  removePointerupListener(listener: (evt: PointerEvent) => void): void;
+  setPointermoveListener(listener: (evt: PointerEvent) => void): void;
+  removePointermoveListener(listener: (evt: PointerEvent) => void): void;
+  setKeydownListener(listener: (evt: KeyboardEvent) => void): void;
+  removeKeydownListener(listener: (evt: KeyboardEvent) => void): void;
+  setKeyupListener(listener: (evt: KeyboardEvent) => void): void;
+  removeKeyupListener(listener: (evt: KeyboardEvent) => void): void;
+  getParentProperty(name: string): unknown;
+  setLabel(value: string): void;
+  setDisabled(value: boolean): void;
+  setPosition(value: SplitViewPanelPosition): void;
+  setOrientation(value: SplitViewOrientation): void;
+  setOpen(value: boolean): void;
+  setGrabbed(value: boolean, orientation: SplitViewOrientation): void;
+  getContentSize(orientation: SplitViewOrientation): number;
+  setContentSize(value: number): void;
+  setValue(value: number): void;
+  getAvailableSpace(orientation: SplitViewOrientation, position: SplitViewPanelPosition): number;
+  getSiblingContentSize(): number;
+  setSiblingContentSize(value: number): void;
+}
+
+export class SplitViewPanelAdapter extends BaseAdapter<ISplitViewPanelComponent> implements ISplitViewPanelAdapter {
+  private _root: HTMLElement;
+  private _handle: HTMLElement;
+  private _icon: IIconComponent;
+  private _content: HTMLElement;
+  private _parent?: ISplitViewComponent;
+
+  constructor(component: ISplitViewPanelComponent) {
+    super(component);
+    this._root = getShadowElement(component, SPLIT_VIEW_PANEL_CONSTANTS.selectors.ROOT);
+    this._handle = getShadowElement(component, SPLIT_VIEW_PANEL_CONSTANTS.selectors.HANDLE);
+    this._icon = getShadowElement(component, SPLIT_VIEW_PANEL_CONSTANTS.selectors.ICON) as IIconComponent;
+    this._content = getShadowElement(component, SPLIT_VIEW_PANEL_CONSTANTS.selectors.CONTENT);
+  }
+
+  public initialize(): void {
+    // Set the parent split view
+    const parent = this._component.parentElement;
+    if (parent?.tagName.toLowerCase() === SPLIT_VIEW_CONSTANTS.elementName) {
+      this._parent = parent as ISplitViewComponent;
+    }
```

Comment:
I know this will probably break if the `<forge-split-view>` is not a direct parent, but just wanted to mention we have a utility in the forge core lib called `walkUpUntil()` that we could use to locate the parent. You likely already know about it, just wanted to mention it:
https://github.com/tyler-technologies-oss/forge-core/blob/9251d3747c1711f8fde08ee436fd45fef5b8498d/src/utils/dom-utils.ts#L629-L643